### ISSUE TITLE
Mega and Z-move shop

### ISSUE THREAD

**Issue by Firemelon66** [SmashMC-Development/Suggestions#955]
**Describe the suggestion**
Should there be a Megastone/Z-Moves shop

**Which servers does this
Every Server

I think it should be fine few people will say its op or sum but i think for the right price it should be ok since it can be most of the times quite a hunt to sit there getting them

**Comment by manlybro2002** [comment]
I like this, more ppl would prob use megas with this. Shouldnt have any big impact on megastone crates since you can get em free from lvl 100 skill rewards or a  lucky vote crate.

**Comment by borkborkdog** [comment]
As I see how this is desirable I don't think this is a good idea because then the market for these megastones will just decrease. For example, stones such as aerodactylite and aggronite are quite rare and not many people have it (on kyogre), adding a megastone shopkeeper in the mart will make it so that everyone has access to the stone and will make it easier for them to get it.

I agree with the z-move mart thought. It was there before reset and instead of having to grind pokeloots that are found in the wild you could pay 30-40k for em and I believe that would be a fair price. In addition, non competitive players most likely will not be using due to their lack of battle knowledge. Finally, as proven from last reset the z move mart was not game breaking.

**Comment by manlybro2002** [comment]
Cant the prices on megastones have the same treatment as rare candies? where mart represents the max price so ppl can still profit from sellin megastones they find.

I guess megastone eco is different for kyogre? on kyurem the megastone prices are 60k-80k and legendary megas are 100k+, although there are a few megas that are worth less like around 20k-40k cos they arent that good.

**Comment by borkborkdog** [comment]
That's the issue, different servers = different economies. On kyogre some stones will sell for 400k+ while other sell for 80k. The minimum we sell for I think is around 50k and the max is around 450k and for people like me who have collected all the mega stones it will come out to be useless in the end as people won't want to buy them from me but instead from mart.

**Comment by M1n0j** [comment]
The Z-Move shop keeper does currently exist in the spawning marts in Ultra Space.

However I do believe having a shop keeper for mega stones as bork said could ruin the eco on some servers due to them all having their own sell prices per server. Eg Mega Char X on Kyurem could be 50k and could be 500k on Ray. It would also make MegaStone Crates useless and give them no point.

**Comment by jantjewieger** [comment]
Yeah i do not think a mega stone shopkeeper should be added. It would drastically take away the value of mega stones (and for the server it would drastically decrease the value of mega stone crate keys)

**Comment by morrow12** [comment]
I agree with wieger, Z crystals would be nice tho priced at 20k per typings, and 50k for specific mon z crystals (Kommonium Z}
Mikium Z) etc.

**Comment by XITACE** [comment]
Totally aggreed
